controllers/user.js

```diff
@@ -689,6 +689,49 @@ async function getRemoveMoreTimePage(req, res) {
   });
 }
 
+function alphabetizeByBroker(data) {
+  return data.sort(function (a, b) {
+    if (a.broker < b.broker) {
+      return -1;
+    }
+    if (a.broker > b.broker) {
+      return 1;
+    }
+    return 0;
+  });
+}
+
+function sortRemovalData(removalData) {
+  const completeItems = removalData.filter((removalItem) => {
+    return removalItem.status === REMOVAL_STATUS["COMPLETE"].id;
+  });
+
+  alphabetizeByBroker(completeItems);
+
+  const blockedItems = removalData.filter((removalItem) => {
+    return (
+      removalItem.current_step === JS_CONSTANTS.REMOVAL_STEP["BLOCKED"].code
+    );
+  });
+
+  alphabetizeByBroker(blockedItems);
+
+  const activeItems = removalData.filter((removalItem) => {
+    return (
+      removalItem.status !== REMOVAL_STATUS["COMPLETE"].id &&
+      removalItem.current_step !== JS_CONSTANTS.REMOVAL_STEP["BLOCKED"].code
+    );
+  });
+
+  alphabetizeByBroker(activeItems);
+
+  const sortedData = [...completeItems, ...activeItems, ...blockedItems];
+
+  console.log("sorted", sortedData);
```

Comment:
nit: Not sure if we want to add an issue to remove the debugging `console.log` statements before going to production.